### ISSUE TITLE
Define the DB Schema for `finalize`d LN Swaps

### ISSUE THREAD

**Issue by D4nte** [comit-network/comit-rs#2177]
## Problem

We learn new information about the other party during the communication phase and before the execution phase, concretely, identities and in the case of Bob, also the secret hash.

This information needs to be persisted in the database.

## Goal

Design a DB schema for the information we learn during the communication phase.
Aim for the design to be reusable across several swaps.

## Out of scope

The DB schema for `created`d swap should already be done in https://github.com/comit-network/comit-rs/issues/2173
Saving and retrieving from the new tables is not part of this issue.

## Recommendation

KISS.

Keep in mind that this issue was unblocked from https://github.com/comit-network/comit-rs/issues/2173 with the idea that we use separate tables for this.

**Comment by tcharding** [comment]
Why is this blocked on https://app.zenhub.com/workspace/o/comit-network/comit-rs/issues/2170?

**Comment by D4nte** [comment]
@luckysori to create ticket that tracks the "swap spawn" on the core for HALight. Such ticket will block this ticket as it will define the data that we need for HALight (and hence, this is the data to be saved in the DB).

Please ping @tcharding once such ticket is created so he can update his dependencies/tickets.

**Comment by thomaseizinger** [comment]
Removing the blocker on https://github.com/comit-network/comit-rs/issues/2173 because we can design these tables independently from the one where the created swaps are stored.

**Comment by tcharding** [comment]
At some stage the `start_of_swap` will most likely need to be created by the database i.e., with a table row: `at DATETIME DEFAULT CURRENT_TIMESTAMP` in either of the tables for 'created' or 'finalized' swaps.  Adding comment here to keep track of it.

`start_of_swap` is mentioned here: https://github.com/comit-network/comit-rs/issues/2408

And this should probably be either done in this ticket or in the [created](https://github.com/comit-network/comit-rs/issues/2173) side.

**Comment by thomaseizinger** [comment]
Obsoleted by https://github.com/comit-network/comit-rs/issues/2485.
